superduperdb/components/model.py

```diff
@@ -719,6 +719,37 @@ def __call__(self, outputs: t.Optional[str] = None, **kwargs):
             identifier=outputs,
         )
 
+    def to_listener(
+        self,
+        key: ModelInputType,
+        select: CompoundSelect,
+        identifier='',
+        predict_kwargs: t.Optional[dict] = None,
+        db: t.Optional["Datalayer"] = None,
```

Comment:
Let us remove this parameter.

The reason is that I want this `db.add` to be central usage.

Reply:
Done